Batteries/Lean/EStateM.lean

```diff
@@ -33,7 +33,7 @@ end Result
 @[simp] theorem run_map (f : α → β) (x : EStateM ε σ α) :
     (f <$> x).run s = (x.run s).map f := rfl
 
-@[ext] theorem ext {ε σ α : Type u} (x y : EStateM ε σ α) (h : ∀ s, x.run s = y.run s) : x = y := by
+@[ext] theorem ext {ε σ α} (x y : EStateM ε σ α) (h : ∀ s, x.run s = y.run s) : x = y := by
```

Comment:
```suggestion
@[ext] theorem ext {ε σ α} {x y : EStateM ε σ α} (h : ∀ s, x.run s = y.run s) : x = y := by
```

Reply:
What do you think about using `⦃⦄` binders for the `ext` lemmas? There's some precedent for that, as it means that these can be type-cast as `Function.Injective`?

Reply:
In what sense does this math `Function.Injective`? I see `(fun s => x.run s) = (fun s => y.run s) -> x = y` but that doesn't seem very useful. I'm probably missing something.

Reply:
In precisely that sense. Agreed that it isn't very useful here (which is why I took your suggestion), but it's useful in some cases and mostly harmless in the rest, so perhaps sensible as a general pattern to consider in the future.